hal_st/stm32fxxx/AdcDmaMultiChannelStm.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2024 Koninklijke Philips N.V.
```

Comment:
```suggestion
```

Reply:
The file being in our archive is enough notice that it is copyrighted, so we leave out these individual lines